addons/account/models/chart_template.py

```diff
@@ -191,11 +191,11 @@ def load_for_current_company(self, sale_tax_rate, purchase_tax_rate):
                 raise UserError(_('Could not install new chart of account as there are already accounting entries existing.'))
 
             # delete accounting properties
-            prop_values = ['account.account,%s' % (account_id,) for account_id in existing_accounts.ids]
+            prop_values = existing_accounts.ids
             existing_journals = self.env['account.journal'].search([('company_id', '=', company.id)])
             if existing_journals:
-                prop_values.extend(['account.journal,%s' % (journal_id,) for journal_id in existing_journals.ids])
-            accounting_props = self.env['ir.property'].search([('value_reference', 'in', prop_values)])
+                prop_values.extend(existing_journals.ids)
+            accounting_props = self.env['ir.property'].search([('value_integer', 'in', prop_values)])
```

Comment:
Maybe add the condition `('type', '=', 'many2one')` in the domain.
On can also simplify `prop_values`:
```python
existing_journals = ...
prop_values = existing_accounts.ids + existing_journals.ids
```